src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Component test is failing for this file. You probably need to delete the newline at the end of this file.

Reply:
I tried deleted the line but it doesn't work. I also tried deleting the file and then upload an empty file but Github doesn't allow that. Do you have any suggestions for this? If not, I will just make a new branch and commit again

Reply:
This has to be an issue with your editor and/or git configuration. Some editors ensure you have a trailing newline for every single file. I opened it and can see that it has 2 empty lines. An excessive LF (`\n` newline character) is added at the end of your empty file.

Here is a safe way to create a empty file (assuming using Linux):
1. Delete `saving-complete-modal.component.scss`
2. Execute `touch saving-complete-modal.component.scss`

This will guarantee that an empty file (without any content or newline characters) being created.

If this still does not work, it must be an issue with your git configuration, which adds a newline at the moment of commit. You can search online for more information on this.